### ISSUE TITLE
[Feature request] Xperia Skin

### ISSUE THREAD

**Issue by iWARR** [dslul/openboard#9]
First of all I want to thank you for your FOSS keyboard.
There is nothing to say how much it's important for the privacy nowadays.

I've searched for such project for a long time; in the past there was no other option than taking a good keyboard and debloating it.

And my choice was "Xperia keyboard" that have an exellent UIX design (the best until today, IMO), that's why many people love it so much.

The development was discontinued long time ago, so users have only the last version 8.1.A.0.12 and many mods based on this version.

So I want propose you to reproduce Xperia UI as additional skin, taking attention to the main nuances have made this keyboard so popular.
I'm not sure whether you interesting in that, but let me show you some pics and how design is really functional, intelligent and accurate.

### As a references:

1. Shots of the real debloated, redesigned and symbols-extended mod

![mod B  - EN](https://user-images.githubusercontent.com/33152262/72212542-f7e6d500-34d5-11ea-8cd1-08f37cbc0339.png)
![Xperia Keyboard](https://user-images.githubusercontent.com/33152262/72212544-fd441f80-34d5-11ea-9095-39ee7df35f54.png)
### 
2. A new concept (not realized)

- More darken background to provide more buttons contrast
- Bottom line: new buttons compound (size, margins, order)
- Right/Left buttons added for cursor moving: [<] [Space] [>]

![Xperia Keyboard - NEW Concept](https://user-images.githubusercontent.com/33152262/72212547-07661e00-34d6-11ea-84d4-0ff2e133edbd.png)
This shot is (1:1), so you can download and look at it more closely.

May be you will like some ideas.

**Comment by dslul** [comment]
I had this keyboard on my old Z3, I like this design a lot, I think I'll add it. I was also thinking to add some sort of theme customizer, so people could adjust colors as they like.

**Comment by iWARR** [comment]
Great!

**Some additional notes:**

- Xperia buttons design based on "shadow-effects", so probably this skin will appear different from common "Theme customizer" themes (technically). I don't know if procedural shadows are possible, or raster parts should be mixed within vector/svg parts. You are dev (I'm not), so you should know better. 

- Take into account the buttons margins/padding (vertical and horizontal) of these examples. Tweaking this in the mods above have made an overall look more compact (better usability).

- A "numbers" row on the top (always visible) was added and the height of this row was slightly decreased (1/2...2/3 or so)

- Optional: I don't know if custom fonts is a possible feature for the "Theme customizer" and how much hard to realize this. I know that usually keyboards use a global system font. (On my shots you can see "Exo2", an exellent font for any UI).

Thank you for your time and kind attention.
